draft-ietf-tls-esni.md

```diff
@@ -934,6 +937,17 @@ IANA is requested to create an entry, echo_required(121) in the
 existing registry for Alerts (defined in {{!RFC8446}}), with the
 "DTLS-OK" column being set to "Y".
 
+# ECHOConfig Extension Guidance {#config-extensions}
+
+Any future information or hints that influence the outer ClientHello SHOULD be
+specified as ECHOConfig extensions. This is primarily because the outer ClientHello
+exists only in support of ECHO. Namely, it is both an envelope for the encrypted
+inner ClientHello and enabler for authenticated key mismatch signals
+(see {{server-behavior}}). In contrast, the inner ClientHello is the true ClientHello
+used upon ECHO negotiation. Any information that influences the inner ClientHello,
+if needed, therefore belongs elsewhere, such as the corresponding HTTPSSVC
+record {{!HTTPSSVC}}.
+
```

Comment:
I'm not sure there is a need to be this directive here. Why not leave this for future work.

Also, hypothetically, suppose that the extension was like "make the nonce twice as long"

Reply:
That's fair (and a good point). It is a bit... restrictive. I just removed the sentence.